src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -0,0 +1,162 @@
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    public void whenCreateNewEventTypePersistItInTheDatabase() throws Exception {
+        EventType eventType = buildEventType();
+
+        repository.saveEventType(eventType);
+
+        final int rows = template.queryForObject("SELECT count(*) FROM zn_data.event_type", Integer.class);
+        assertThat("Number of rows should encrease", rows, equalTo(1));
+
+        SqlRowSet rs = template.queryForRowSet("SELECT et_name, et_event_type_object FROM zn_data.event_type");
+        rs.next();
+
+        assertThat("Name is persisted", rs.getString(1), equalTo("event-name"));
+
+        ObjectMapper mapper = (new NakadiConfig()).jacksonObjectMapper();
+        EventType persisted = mapper.readValue(rs.getString(2), EventType.class);
+
+        assertThat(persisted.getCategory(), equalTo(eventType.getCategory()));
+        assertThat(persisted.getName(), equalTo(eventType.getName()));
+        assertThat(persisted.getEventTypeSchema().getType(), equalTo(eventType.getEventTypeSchema().getType()));
+        assertThat(persisted.getEventTypeSchema().getSchema(), equalTo(eventType.getEventTypeSchema().getSchema()));
+    }
+
+    @Test(expected = DuplicatedEventTypeNameException.class)
+    public void whenCreateDuplicatedNamesThrowAnError() throws Exception {
+        EventType eventType = buildEventType();
+
+        repository.saveEventType(eventType);
+        repository.saveEventType(eventType);
+    }
+
+    @Test
+    public void whenEventExistsFindByNameReturnsSomething() throws NakadiException, DuplicatedEventTypeNameException, NoSuchEventTypeException {
+        EventType eventType = buildEventType();
+
+        repository.saveEventType(eventType);
+
+        EventType persistedEventType = repository.findByName(eventType.getName());
+
+        assertThat(persistedEventType, notNullValue());
+    }
+
+    @Test(expected = NoSuchEventTypeException.class)
+    public void whenEventDoesntExistsFindByNameReturnsNothing() throws NakadiException, NoSuchEventTypeException {
+        EventType eventType = repository.findByName("inexisting-name");
+    }
+
+    @Test
+    public void whenUpdateExistingEventTypeItUpdates() throws NakadiException, DuplicatedEventTypeNameException, IOException, NoSuchEventTypeException {
+        EventType eventType = buildEventType();
+
+        repository.saveEventType(eventType);
+
+        eventType.setCategory("new-category");
+
+        repository.update(eventType);
+
+        SqlRowSet rs = template.queryForRowSet("SELECT et_name, et_event_type_object FROM zn_data.event_type");
+        rs.next();
+
+        assertThat("Name is persisted", rs.getString(1), equalTo("event-name"));
+
+        ObjectMapper mapper = (new NakadiConfig()).jacksonObjectMapper();
+        EventType persisted = mapper.readValue(rs.getString(2), EventType.class);
+
+        assertThat(persisted.getCategory(), equalTo(eventType.getCategory()));
```

Comment:
This is a duplicated "full object" comparison. Related to https://github.com/zalando/nakadi/pull/89/files#r52033466